benchmark/utils/DriverUtils.swift

```diff
@@ -380,16 +363,16 @@ func runBench(_ test: Test, _ c: TestConfig) -> BenchResults? {
     print("Running \(test.name) for \(c.numSamples) samples.")
   }
 
+  test.setUpFunction?()
+
   let sampler = SampleRunner()
   for s in 0..<c.numSamples {
     let time_per_sample: UInt64 = 1_000_000_000 * UInt64(c.iterationScale)
 
     var scale : UInt
     var elapsed_time : UInt64 = 0
     if c.fixedNumIters == 0 {
-      test.setUpFunction?()
       elapsed_time = sampler.run(test.name, fn: testFn, num_iters: 1)
```

Comment:
Why are you changing this in this commit? This has nothing to do with the change you are trying to make. It shouldn't be in this PR.